cypress/integration/models/applicationinventory/applicationinventory.ts

```diff
@@ -231,6 +231,10 @@ export class ApplicationInventory {
             });
     }
 
+    protected performRowAction(buttonName: string): void {
+        cy.get(tdTag).contains(this.name).parent(tdTag).siblings(tdTag).find(buttonName).click();
+    }
+
```

Comment:
can we move this method to utils . It looks generic

Reply:
I had placed this in utils earlier, but for two models, there is a slight modification, where extra parent tag is needed. I will try to find best possible solution to keep this once.